README.md

```diff
@@ -40,6 +40,8 @@ Optional:
 * `thrift_sasl>=0.2.1` for hive and/or Kerberos support. This also requires a SASL
    library to be installed on your system - see [System SASL](#system-sasl)
 
+* `kerberos` for Kerberos over HTTP support.
```

Comment:
What version of the kerberos module does this require? OK if it's just the most recent release but we should document.

What Python versions did you test with? Would be good to test with python 2.7 and a recent Python 3 version at least.

Reply:
We should at minimum test with with CDSW/CML Python 2 and 3 versions that are listed below. This is to ensure that this feature works seemlessly in CDSW/CML
https://docs.cloudera.com/documentation/data-science-workbench/latest/topics/cdsw_release_notes.html#concept_mxd_jmg_1kb__section_e14_hcr_jkb

Reply:
I've tested the patch with python 2.6.9, 2.7.17 and 3.6.9.

I've only tested it with the most recent version of the kerberos module, which is 1.3.0, so I've added 1.3.0 to the README file and to setup.py.